### PR TITLE
Refresh onmount

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

- fixes the sidebar, since it needs to refresh after everything is mounted so it will accurately reflect the proper state. This refresh got removed in another PR, this just puts it back in.

**Testing**

- updated

**Progress**

- ready to go

### CODE REVIEW

**Comment by MorganLove** [comment]
@jonahkagan This is only based on #749 to avoid hassles with the types. This can be merged to main after the other is merged.

File: client/src/components/HomeScreen.test.tsx
Comment (jonahkagan) [inline client/src/components/HomeScreen.test.tsx:122]: any way this chunk can go inside the definition `setupScreenCalls`? It looks like it's repeated three times in this PR

**Review by jonahkagan — APPROVED** [review]

File: client/src/components/HomeScreen.test.tsx
Comment (MorganLove) [reply client/src/components/HomeScreen.test.tsx:122]: @jonahkagan I was waffling on that. Here's why:

```
      ...setupScreenCalls,
      aaApiCalls.getSettings(auditSettings.blank),
      aaApiCalls.getJurisdictionFile,
```
You see this pattern about as much as you see this pattern:

```
      ...setupScreenCalls,
      aaApiCalls.getJurisdictionFile,
```

You also may or may not have the `getRounds` bit after, too. So basically, if I'm going to include even one more line, whether it's `getSettings` or `getJurisdictionFile`, I'd need to create two separate versions of `setupScreenCalls` or have some other complexity to include or exclude the settings endpoint. This looked to me like the simplest solution while still avoiding as much duplication as possible.

Comment (jonahkagan) [reply client/src/components/HomeScreen.test.tsx:122]: cool, seems like this is fine then. hopefully we can reduce the number of API calls as a better solution :)

**Pull request merged**
